test/test.js

```diff
@@ -14,6 +14,8 @@ import puppeteer from 'puppeteer';
 
 		const element = await page.$('p');
 		message = await page.evaluate(element => element.textContent, element);
+	}catch(exception){
+		console.error('[test.js]: ', exception);
```

Comment:
```suggestion
```

We don’t want to catch the exception, because we want the script to crash on error (in order to fail the test). I’m assuming this might’ve been left by accident while you were debugging.

Reply:
Sure! Done https://github.com/disney/meteor-base/pull/134/commits/4ac72397d86604c3de7ad0149ecc655409dae322 . Thanks!